understanding/22/accessible-authentication-minimum.html

```diff
@@ -68,16 +68,17 @@ <h2>In brief</h2>
     <section id="intent">
         <h2>Intent of Accessible Authentication</h2>
 
-        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method to log in. Most Web sites rely on usernames and passwords for logging in. Memorizing or transcribing a username, password, or one-time verification code places a very high or impossible burden upon people with certain cognitive disabilities.</p>
+        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method for users to authenticate -- to prove they are authentic account holders. Most Web sites rely on usernames and passwords for logging in, the most common form of authentication. Yet memorizing or transcribing a username, password, or one-time verification code places a very high or impossible burden upon people with certain cognitive disabilities.</p>
 
         <p>While Web sites can use the recognition of objects or of non-text content provided by the user to meet this Success Criterion, such techniques do not fully support the cognitive accessibility community and should be avoided if possible. Refer to <a href="accessible-authentication-enhanced">Accessible Authentication (Enhanced)</a> for guidance to be more inclusive and accessible.</p>
 
+        <p>This Success Criterion does <em>not</em> address creation of a username or intitiation of an account.  For many Web sites, establishing an initial username is not significantly different than logging in with that username.  However, for many other Web sites, the processes are quite different.  Moreover, the means used for creating an account vary more than the means used for authentication.  This is because initial validation may require more due diligence with assuring the identity of a new user.  Another substantive difference between account creation and subsequent user authentication using that account is that a third party, human or automated, is often required in the former.</p>
```

Comment:
```suggestion
        <p>This Success Criterion does <em>not</em> address creation of a username or intitiation of an account.  For many Web sites, establishing an initial username is not significantly different from logging in with that username.  However, for many other Web sites, the processes are quite different.  Moreover, the means used for creating an account vary more than the means used for authentication.  This is because initial validation may require more due diligence with assuring the identity of a new user.  Another substantive difference between account creation and subsequent user authentication using that account is that a third party, human or automated, is often required in the former.</p>
```

Reply:
I personally don't buy this argument. It sounds like a retrospective rationalisation. And I'd argue the claim that registration involves a third party only applies to a very small number of cases/sites, so it's extremely weak as a rationale.

The whole argument about limiting to authentication (as in "logging in") was, if I recall correctly, that it was all about users not needing to remember things. This was then expanded to also lump in captchas etc, but primarily this was an SC aimed at not forcing users to have an accurate memory of things like passwords that they had previously set. So clearly it doesn't apply to registration in that context, because at that point the users haven't provided a password or similar YET so they're not being asked to *remember* things.

Reply:
I didn't write it, I just made it more readable, and tried to address some comments in the issue :) I'll attempt to do a pass with your comment in mind, but if you have suggested changes, please go for it!

Reply:
> I didn't write it

Yes sorry, that comment was meant to be not directed at you (I originally had misfiled it to your commit, but then closed that and posted it back to the main PR here). So yes, it's directed at @bruce-usab

Reply:
LMK if my new changes are more inline with what you are looking for.